src/main/java/com/alkemy/ong/mapper/ContactMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.alkemy.ong.mapper;
+
```

Comment:
I believe this is part of my ticket to solve the endpoint, yours mentions Entity, Repository and Service only